.gitignore

```
@@ -68,3 +69,4 @@ examples/mediadevices/public/helpers.js
 examples/networkquality/public/helpers.js
 examples/reconnection/public/helpers.js
 examples/screenshare/public/helpers.js
+examples/localmediacontrols/public/helpers.js
```

@PikaJoyce Please add a new line here:
```suggestion
examples/localmediacontrols/public/helpers.js

```